Rubberduck.Deployment/InnoSetup/Includes/German.CustomMessages.iss

```diff
@@ -5,21 +5,22 @@ German.RegisterAddin=VBE AddIn Registrierung reparieren
-German.WillInstallForCurrentUser=Das Rubberduck VBE-AddIn wird für den aktuellen Nutzer verfügbar gemacht und keine Administratorprivilegien benötigen.
-German.UninstallOldVersionPrompt=Es scheint eine vorige Version von Rubberduck, als Anwendung für %s, installiert zu sein, die Deinstalliert werden sollte. Fortfahren?
-German.UninstallOldVersionFail=Konnte vorige Version nicht deinstallieren. Versuchen Sie, diese über die Systemsteuerung zu deinstallieren und starten Sie den Installer erneut.
-German.WarnInstallPerUserOverEveryone=Es scheint eine für alle Benutzer installierte Version von Rubberduck vorzuliegen. Eine installation nur für diesen Benutzer wird Vorrang vor der originalen Installation haben. Trotzdem fortfahren?
-German.ElevationRequestFailMessage=Konnte vorige Version von Rubberduck nicht deinstallieren. Es kann sein, dass Sie diese manuell deinstallieren müssen, bevor Sie die neue Version installieren können.
\ No newline at end of file
+German.WillExecuteAdminInstall=Das Rubberduck AddIn wird für alle Benutzer verfügbar sein.%n%nACHTUNG: Jeder Benutzer muss selbst das AddIn registrieren, da dies eine Nutzereinstellung ist%nund nicht für alle Benutzer direkt eingerichtet werden kann.%n%nNach der Installation wird eine Verlinkung mit dem Namen "VBE AddIn Registrierung reparieren" im%nProgrammordner von Rubberduck für alle Nutzer verfügbar sein.
+German.WillLaunchAdminInstall=Die Installation wird nach Administratorprivilegien zum Installieren für alle Nutzer fragen.%nAnschließend wird die Installation fortgesetzt und das AddIn für diesen Nutzer registriert.
+German.WillInstallForCurrentUser=Das Rubberduck VBE-AddIn wird für den aktuellen Nutzer verfügbar gemacht und es werden keine Administratorprivilegien benötigen.
+German.UninstallOldVersionPrompt=Es scheint eine vorherige Version von Rubberduck als Anwendung für %s installiert zu sein, die deinstalliert werden sollte. Fortfahren?
+German.UninstallOldVersionFail=Konnte vorherige Version nicht deinstallieren. Versuchen Sie, diese über die Systemsteuerung zu deinstallieren und starten Sie den Installer erneut.
+German.WarnInstallPerUserOverEveryone=Es scheint eine für alle Benutzer installierte Version von Rubberduck derzeit auf dem System vorzuliegen. Eine Installation nur für diesen Benutzer wird Vorrang vor der originalen Installation haben. Trotzdem fortfahren?
+German.ElevationRequestFailMessage=Konnte vorherige Version von Rubberduck nicht deinstallieren. Es kann sein, dass Sie diese manuell deinstallieren müssen, bevor Sie die neue Version installieren können.
+German.UninstallOldVersionNotFound=Das Deinstallationsprogramm konnte nicht gefunden werden. Möchten Sie fortfahren und dennoch installieren? Dies kann potentiell eine existierende Installation überschreiben und unter Umständen zu duplizierten Einträgen in der Liste Programme deinstallieren oder ändern führen. Trotzdem fortfahren? 
```

Comment:
I would put the name of the program list in quotes to avoid having to weave it into the sentence flow. As it is right now, that flow is horrible

Reply:
@bclothier Are there any restrictions on using quotes in InnoSetup resources?

Reply:
Based on [this line](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Deployment/InnoSetup/Includes/English.CustomMessages.iss#L19), it should work just fine. The [documentation](https://jrsoftware.org/ishelp/index.php?topic=messagessection) seems to imply only `%` character has special meaning. In other section such as `[Run]`, the quotes has to be doubled up but I guess that doesn't apply here.